lib/dat-worker-pool/runner.rb

```diff
@@ -78,27 +97,44 @@ def build_worker
     # ensures if the hard shutdown is raised and not rescued (for example, in
     # the workers ensure), then it won't cause the forced shutdown to end
     # prematurely
-    def force_shutdown(timeout, backtrace)
-      error = ShutdownError.new("Timed out shutting down (#{timeout} seconds).")
-      error.set_backtrace(backtrace)
+    def forced_shutdown(orig_exception, timeout, &backtrace_proc)
```

Comment:
@jcredding did you mean to rename it to `forced_shutdown`?  Just curious - I don't necessarily have a problem with this.